kge/job/train.py

```diff
@@ -78,15 +81,22 @@ def __init__(
         self.config.check("train.trace_level", ["batch", "epoch"])
         self.trace_batch: bool = self.config.get("train.trace_level") == "batch"
         self.epoch: int = 0
-        self.valid_trace: List[Dict[str, Any]] = []
-        valid_conf = config.clone()
-        valid_conf.set("job.type", "eval")
-        if self.config.get("valid.split") != "":
-            valid_conf.set("eval.split", self.config.get("valid.split"))
-        valid_conf.set("eval.trace_level", self.config.get("valid.trace_level"))
-        self.valid_job = EvaluationJob.create(
-            valid_conf, dataset, parent_job=self, model=self.model
-        )
+        self.is_forward_only = forward_only
+
+        if not self.is_forward_only:
+            self.model.train()
```

Comment:
otherwise need to set self.model.eval()? Or is this done outside in the surrounding evaluation job?

Reply:
Done in the eval job